### PR TITLE
Improve O to R

### PR DESCRIPTION
**Pull request by GabrielRavier**
That was fun

### CODE REVIEW

File: src/PixTone.cpp [hunk 1]
```diff
@@ -139,13 +139,13 @@ BOOL MakePixelWaveData(const PIXTONEPARAMETER *ptp, unsigned char *pData)
...
-			* envelopeTable[d]
-			/ 64
-			+ 128;
+				   * ptp->oMain.top
+				   / 64
+				   * (gWaveModelTable[ptp->oVolume.model][c] * ptp->oVolume.top / 64 + 64)
+				   / 64
+				   * envelopeTable[d]
+				   / 64
+				   + 128;
```
Comment (Clownacy) [inline src/PixTone.cpp]: Why?

File: src/Organya.cpp [hunk 2]
```diff
@@ -19,8 +19,8 @@
 #define MAXMELODY 8
 #define MAXDRAM 8
 
-SOUNDBUFFER* lpORGANBUFFER[8][8][2] = {NULL};
-SOUNDBUFFER** lpDRAMBUFFER = &lpSECONDARYBUFFER[0x96];
+SOUNDBUFFER *lpORGANBUFFER[8][8][2] = { NULL };
```
Comment (Clownacy) [inline src/Organya.cpp]: I prefer initialiser lists without spaces on each side.

File: src/PixTone.cpp (hunk 1 above)
Comment (GabrielRavier) [reply src/PixTone.cpp]: Mostly because it makes the formatting look better

Comment (Clownacy) [reply src/PixTone.cpp]: I don't see how two indents and three spaces looks better than one indent.

File: src/Organya.cpp (hunk 2 above)
Comment (GabrielRavier) [reply src/Organya.cpp]: aaaaa half of them are with spaces and half are without. I'll revert them to no spaces I guess. Btw by "initialiser list", do you mean "any initialiser with braces" or "initializers with multiple elements" ?

File: src/PixTone.cpp (hunk 1 above)
Comment (GabrielRavier) [reply src/PixTone.cpp]: It's to align the operators with the initializers

Comment (Clownacy) [reply src/PixTone.cpp]: That doesn't seem to work if the indent size isn't 4. GitHub seems to be displaying at size 8 for me, and they're not aligned.

Comment (GabrielRavier) [reply src/PixTone.cpp]: I'll try aligning it by indenting with tabs until the normal code placement and then indenting with spaces (complicated but you'll get it when I push)

Comment (Clownacy) [reply src/PixTone.cpp]: Good idea.

Comment (GabrielRavier) [reply src/PixTone.cpp]: Probably fixed it for all indentation levels

File: src/Organya.cpp (hunk 2 above)
Comment (Clownacy) [reply src/Organya.cpp]: Any initialiser with braces.

Comment (GabrielRavier) [reply src/Organya.cpp]: Ok, fixing that

Comment (GabrielRavier) [reply src/Organya.cpp]: Fixed

**Pull request merged**